app/services/import_user.rb

```diff
@@ -43,4 +41,12 @@ def self.create(org_id)
   def self.generate_pin
     "#{rand(10)}#{rand(10)}#{rand(10)}#{rand(10)}"
   end
+
+  def self.bot_phone_number(org_id)
+    # Losslessly compresses an organisation UUID value into a url-safe base64 representation.
+    # This is a bit more compact than stuffing a UUID string as a phone number directly. Other
+    # forms of hashing may lead to information loss, doing it this way keeps the collision
+    # probability the same as the organization UUID itself.
+    [[org_id.remove("-")].pack("H*")].pack("m0").tr("+/", "-_")[..21]
+  end
```

Comment:
I know this bit of code is ugly, but given that it's short and used in a contained manner, it felt okay.